embed/whereby-embed.js

```diff
@@ -56,6 +59,21 @@ define("WherebyEmbed", {
     }
     `;
   },
+
+  // Commands
+  toggleCamera(enabled) {
+    const url = new URL(this.room, `https://${this.subdomain}.whereby.com`);
+    if (this.iframe.current) {
+      this.iframe.current.contentWindow.postMessage({ command: "toggle_camera", args: [enabled]}, url.origin);
+    }
+  },
+  toggleMicrophone(enabled) {
+    const url = new URL(this.room, `https://${this.subdomain}.whereby.com`);
+    if (this.iframe.current) {
+      this.iframe.current.contentWindow.postMessage({ command: "toggle_microphone", args: [enabled]}, url.origin);
+    }
```

Comment:
Move the creation of the `url` variable inside the if statement since it's not used outside of it?

```suggestion
  toggleCamera(enabled) {
    if (this.iframe.current) {
      const url = new URL(this.room, `https://${this.subdomain}.whereby.com`);
      this.iframe.current.contentWindow.postMessage({ command: "toggle_camera", args: [enabled]}, url.origin);
    }
  },
  toggleMicrophone(enabled) {
    if (this.iframe.current) {
      const url = new URL(this.room, `https://${this.subdomain}.whereby.com`);
      this.iframe.current.contentWindow.postMessage({ command: "toggle_microphone", args: [enabled]}, url.origin);
    }
```

Reply:
Agree. Will fix 👍

Reply:
Fixed